ksql-benchmark/src/main/java/io/confluent/ksql/benchmark/SerdeBenchmark.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.benchmark;
```

Comment:
Perhaps I'm late to the game as far as context goes, but what unique aspect of KSQL are we actually testing here and what's the motivation to introduce these benchmarks? I may be missing something, but it looks like we use pretty standard SerDe components (from Kafka Connect) and we just have lightweight wrappers around them - would it make sense to have this in Kafka Connect instead of KSQL?